### ISSUE TITLE
Does not redis-clustr support typescript?

### ISSUE THREAD

**Issue by mamingzhi** [gosquared/redis-clustr#23]
I found the redis-clustr is very easy to use than other several node redis cluster libraries. I like it, but I cannot find the "@types/redis-clustr" for my typescript project.

Does not redis-clustr support typescript?

**Comment by simontabor** [comment]
No, there's no typescript support at the moment. Would you be willing to contribute the types?

**Comment by jpike88** [comment]
@simontabor you wrote the code, you're going to know the types better than anyone else. I just took a look at the code and it's not easy to know what types certain functions are using even when reading the comments.
